src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http1.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Test.Common;
+//using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Functional.Tests
+{
+    // This class is dedicated to SocketHttpHandler tests specific to HTTP/1.x.
```

Comment:
If this is just about SocketsHttpHandler, why do we need the abstract base and the single SocketsHttpHandler-related class derived from it?

Reply:
I guess we don't need abstract. I copied structure from Http2 but I think we wanted to run Platform tests once in a while...

Reply:
True, but do we think those tests will reasonably pass against the Platform handlers?

I do agree though that eventually we need to rethink how these tests are organized and simplify where appropriate.

Reply:
I think since this is "should" we do not need to force Platform handlers do it. In this particular case I see it as test for particular behavior more so than general protocol or API correctness.